### DISCUSSION TITLE
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'VoiceClient' object has no attribute 'listen'

### DISCUSSION THREAD

**Discussion by RemiZacharias** [Rapptz/discord.py#5994] [Q&A] [answered]
When using Rapptz trying to use the listen feature, I always get the error:
`discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'VoiceClient' object has no attribute 'listen'`

I use Linux (Google Pixelbook to be exact) and I have downloaded all the additional things linux users need to use the API. At least I'm pretty sure It's all correctly downloaded. It also says that can't find WaveSink and UserFilter in __init__.py whenever I try to do discord.UserFliter or discord.WaveSink.

I believe this has something to do with not having downloaded something correctly but I don't know what it could be

**Comment by Rapptz** [comment, marked as answer]
The standard library does not contain these methods. They're available on [a fork somewhere](https://github.com/Rapptz/discord.py/issues/1094) but no official release has these methods and thus won't work.
